storage/container_storage_interface/persistent-storage-csi-ovirt.adoc

```diff
@@ -0,0 +1,27 @@
+[id="persistent-storage-csi-ovirt"]
+= Red Hat Virtualization (oVirt) CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ovirt
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Red Hat Virtualization (oVirt).
+
+Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+
+To create CSI-provisioned PVs that mount to Red Hat Virtualization (oVirt) storage assets, {product-title} installs the oVirt CSI Driver Operator and the oVirt CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
@jsafrane Same question as above - do we need to suggest that the user upgrades the old Operator (if installed)?